### PR TITLE
chore(ui): move users & about nav items

### LINKED ISSUE #17578: Move Org Tab Items
In our recent round of user testing every user expected to find the items currently in the `Organization` tab to be in the user widget.

This issue **_must_** be done with https://github.com/influxdata/quartz/issues/2408

- [x] Move `members` page link into `UserWidget`
- [x] Move `about` page link into `UserWidget`
- [x] Replace all instances of `members` with `users`
- [x] Remove `Organization` item from tree nav

### PR DESCRIPTION
**Pull request by mavarius**
Closes #17578

- Move Org nav items "members" and "about" to Profile nav menu
- Rename "Members" to "Users" in cloud only

image

- [x] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [ ] Tests pass

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
 
 ### UI Improvements
 
+1. [17849](https://github.com/influxdata/influxdb/pull/17849): Move Organization navigation items to user menu.
 1. [17714](https://github.com/influxdata/influxdb/pull/17714): Cloud environments no longer render markdown images, for security reasons.
 1. [17321](https://github.com/influxdata/influxdb/pull/17321): Improve UI for sorting resources
 1. [17740](https://github.com/influxdata/influxdb/pull/17740): Add single-color color schemes for visualizations
```
Comment (hoorayimhelping) [inline CHANGELOG.md:16]: nit: i think we append to the bottom not the top. not that it's a very big deal

```suggestion
1. [17714](https://github.com/influxdata/influxdb/pull/17714): Cloud environments no longer render markdown images, for security reasons.
1. [17321](https://github.com/influxdata/influxdb/pull/17321): Improve UI for sorting resources
1. [17740](https://github.com/influxdata/influxdb/pull/17740): Add single-color color schemes for visualizations
1. [17849](https://github.com/influxdata/influxdb/pull/17849): Move Organization navigation items to user menu.
```

File: ui/src/pageLayout/components/UserWidget.tsx [hunk 2]
```diff
@@ -71,8 +76,43 @@ const UserWidget: FC<Props> = ({
...
           )}
         />
+        <FeatureFlag name="multiUser" equals={true}>
+          <TreeNav.UserItem
+            id="users"
+            label="Users"
+            linkElement={className => (
+              <a
+                className={className}
+                href={`${CLOUD_URL}/organizations/${org.id}${CLOUD_USERS_PATH}`}
```
Comment (hoorayimhelping) [inline ui/src/pageLayout/components/UserWidget.tsx:86]: what's up with `/orgs/` and `/organizations/`?

File: CHANGELOG.md (hunk 1 above)
Comment (mavarius) [reply CHANGELOG.md:16]: Ya, here's the Quartz PR: https://github.com/influxdata/quartz/pull/2530

File: ui/src/pageLayout/components/UserWidget.tsx (hunk 2 above)
Comment (mavarius) [reply ui/src/pageLayout/components/UserWidget.tsx:86]: This is the route we ended up within Quartz, there's discussions ongoing around syncing this and users/members but that's outside the scope of this PR.

**Comment by alexpaxton** [comment]
@mavarius can you throw some screenshots of the changes in here?

**Comment by alexpaxton** [comment]
Might want to rename `Members` to `Users` here: https://github.com/influxdata/influxdb/blob/master/ui/src/members/components/Members.tsx#L65

**Comment by alexpaxton** [comment]
Actually, disregard my last comment

**Comment by mavarius** [comment]
> Make sure to update the router as well

No changes need to be made to the routes, only the UI elements. "Members" is only renamed to "Users" in cloud, not OSS.

**Pull request merged**